concepticondata/conceptlists/Gabelentz-1892-1074.tsv

```diff
@@ -0,0 +1,1075 @@
+Gabelentz-1892-1074-901	901	Hören	to hear	VIII. Verba	C. Leben, Körperfunctionen	1408	HEAR
+Gabelentz-1892-1074-902	902	Kosten	to taste	VIII. Verba	C. Leben, Körperfunctionen	21	TASTE (SOMETHING)
+Gabelentz-1892-1074-903	903	Riechen, schnüffeln	to smell	VIII. Verba	C. Leben, Körperfunctionen	1586	SMELL (PERCEIVE)
+Gabelentz-1892-1074-904	904	Fühlen	to feel	VIII. Verba	C. Leben, Körperfunctionen	1079	FEEL
+Gabelentz-1892-1074-905	905	Berühren, betasten	to touch	VIII. Verba	C. Leben, Körperfunctionen	1892	TOUCH
+Gabelentz-1892-1074-906	906	Den Beischlaf vollziehen	to have intercourse	VIII. Verba	C. Leben, Körperfunctionen	1785	HAVE SEX
+Gabelentz-1892-1074-907	907	Gehen	to go	VIII. Verba	D. Gehen, Kommen u.s.w.	695	GO
+Gabelentz-1892-1074-908	908	Kommen	to come	VIII. Verba	D. Gehen, Kommen u.s.w.	1446	COME
+Gabelentz-1892-1074-909	909	Besuchen	to visit	VIII. Verba	D. Gehen, Kommen u.s.w.	1377	VISIT
+Gabelentz-1892-1074-910	910	Begegnen	to meet	VIII. Verba	D. Gehen, Kommen u.s.w.	1831	MEET
+Gabelentz-1892-1074-911	911	Landen	to land	VIII. Verba	D. Gehen, Kommen u.s.w.	1922	LAND (DESCEND)
+Gabelentz-1892-1074-912	912	Absegeln	to start sailing	VIII. Verba	D. Gehen, Kommen u.s.w.		
+Gabelentz-1892-1074-913	913	Folgen	to follow	VIII. Verba	D. Gehen, Kommen u.s.w.	1085	FOLLOW
+Gabelentz-1892-1074-914	914	Verfolgen	to pursue	VIII. Verba	D. Gehen, Kommen u.s.w.	1047	PURSUE
+Gabelentz-1892-1074-915	915	Erreichen	to arrive at a destination	VIII. Verba	D. Gehen, Kommen u.s.w.	1387	ARRIVE
+Gabelentz-1892-1074-916	916	Umkehren	to turn back	VIII. Verba	D. Gehen, Kommen u.s.w.	581	COME BACK
+Gabelentz-1892-1074-917	917	Schicken, senden	to send	VIII. Verba	D. Gehen, Kommen u.s.w.	1127	SEND
+Gabelentz-1892-1074-918	918	Eintreten	to enter	VIII. Verba	D. Gehen, Kommen u.s.w.	749	ENTER
+Gabelentz-1892-1074-919	919	Treten	to kick	VIII. Verba	D. Gehen, Kommen u.s.w.	1135	KICK
+Gabelentz-1892-1074-920	920	Laufen	to walk	VIII. Verba	D. Gehen, Kommen u.s.w.	1443	WALK
+Gabelentz-1892-1074-921	921	Kriechen	to crawl	VIII. Verba	D. Gehen, Kommen u.s.w.	3456	CRAWL OR CREEP
+Gabelentz-1892-1074-922	922	Klettern	to climb	VIII. Verba	D. Gehen, Kommen u.s.w.	1102	CLIMB
+Gabelentz-1892-1074-923	923	Fliegen	to fly	VIII. Verba	D. Gehen, Kommen u.s.w.	1441	FLY (MOVE THROUGH AIR)
+Gabelentz-1892-1074-924	924	Fliehen	to flee	VIII. Verba	D. Gehen, Kommen u.s.w.	956	FLEE
+Gabelentz-1892-1074-925	925	Verlassen	to leave	VIII. Verba	D. Gehen, Kommen u.s.w.	1752	LEAVE
+Gabelentz-1892-1074-926	926	Aufgehen (die Sonne u.s.w.)	to rise (sun etc.)	VIII. Verba	D. Gehen, Kommen u.s.w.	568	RISE (MOVE UPWARDS)
+Gabelentz-1892-1074-927	927	steigen	to ascend	VIII. Verba	D. Gehen, Kommen u.s.w.	705	GO UP (ASCEND)
+Gabelentz-1892-1074-928	928	Untergehen (desgl.)	to set (sun etc.)	VIII. Verba	D. Gehen, Kommen u.s.w.	160	SET (HEAVENLY BODIES)
+Gabelentz-1892-1074-929	929	hinabsteigen	to descend	VIII. Verba	D. Gehen, Kommen u.s.w.	2014	GO DOWN (DESCEND)
```

Comment:
Not completely sure about the mapping of (hinab)steigen as it's by context limited to sun etc.

Reply:
Is it? I thought that would be covered by SET (HEAVENLY BODIES).

Reply:
It could be about the difference between a heavenly body moving up/down along the firmament and it moving across the horizon.
Probably the concepts (hinab)steigen here refer to people and not to the sun, it just looks strange by their order in the list.

Reply:
By "(hinab)steigen" I meant both "steigen" and "hinabsteigen", sorry if that wasn't clear.

Usually it does refer to a person climbing stairs or a mountain or similar, it's just the context that makes me doubt.

Reply:
Based on the SUBCATEGORY 'Gehen, Kommen u.s.w.", I think the mapping is fine.